docs/dictionary/property/HTMLText.lcdoc

```diff
@@ -28,74 +29,127 @@ write the HTMLText of field "Story" to file myWebFile
 
 The tags translate as follows:
 
 &lt;p&gt; &lt;/p&gt;
-Encloses a line of text. (Blank lines are also enclosed in &lt;p&gt; &lt;/p&gt;.)
+Encloses a line of text. (Blank lines are also enclosed in &lt;p&gt; &lt;/p&gt;.) 
+These properties are represented as attributes of the &lt;p&gt; tag:
+* align="left|center|right" appears if an <textAlign|alignment> has been applied to the line.
+* firstindent="N" (where N is an integer representing number of pixels) appears if the <firstIndent> has been set for the line.
+* leftindent="N" appears if the <leftIndent> has been set for the line.
+* rightindent="N" appears if the <rightIndent> has been set for the line.
+* spaceabove="N" appears if the <spaceAbove> has been set for the line.
+* spacebelow="N" appears if the <spaceBelow> has been set for the line.
+* tabstops="N" appears if <tabStops> have been set for the line.
+* borderwidth="N" if a <borderWidth> has been set for the line.
+* hgrid="true|false" if the <vGrid> property has been set for the line.
+* vgrid="true|false" if the <hGrid> property has been set for the line.
+* dontwrap="true|false" if the <dontWrap> property has been set for the line.
+* padding="N" if the <padding> has been set for the line.
+* hidden="true|false" if the <hidden> property has been set for the line.
+* bgcolor="#NNNNNN" if a <backgroundColor> has been set for the line.
+* bordercolor="#NNNNNN" if a <borderColor> has been set for the line.
+>*Note*: An <HTML>-style color definition consists of a hash mark (#) followed by three 2-digit hexadecimal numbers, 
+one each for red, green, and blue.
 
 &lt;sub&gt; &lt;/sub&gt;
-Encloses text whose textShift is a positive <integer>. (The &lt;sub&gt; tag is not nested for additional levels of subscription: it appears once for a run of subscripted text, regardless of the value of the <textShift>.)
+Encloses text whose <textShift> is a positive <integer>. 
+(The &lt;sub&gt; tag is not nested for additional levels of subscription: 
+it appears once for a run of subscripted text, regardless of the value of the <textShift>.)
+* shift="N" where N is the number of pixels text is subscripted.
 
 &lt;sup&gt; &lt;/sup&gt;
-Encloses text whose textShift is a <negative> <integer>. (The &lt;sup&gt; tag is not nested for additional levels of superscription: it appears once for a run of superscripted text, regardless of the value of the <textShift>.)
+Encloses text whose <textShift> is a <negative> <integer>. (The &lt;sup&gt; tag is not nested for additional levels of superscription: 
+it appears once for a run of superscripted text, regardless of the value of the <textShift>.)
+* shift="-N" where N is the number of pixels text is superscripted.
 
 &lt;i&gt; &lt;/i&gt;
-Encloses text whose textStyle is "italic".
+Encloses text whose <textStyle> is "italic".
+>If a field is set to <HTML> text that includes text wrapped in a &lt;em&gt; &lt;/em&gt;, 
```

Comment:
The `>` character here makes this a blockquote - I *think* we're consistent in prefixing all such blockquote sections with `>*Note:*`. I'm happy for this to either not be blockquote or for it to be a note.